### ISSUE TITLE
GenerateAdapterBindings in ClusterInstall : result is am empty Json string.

### ISSUE THREAD

**Issue by ligadata-haitham** [LigaData/Kamanja#1104]
when running the GenerateAdapterBindings script i get an empty Json string.
this is the command i use : ( ./GenerateAdapterBindings.sh --KamanjaHome /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10 --ClusterConfig /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10/config/ClusterConfig.json --OutputFile /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10/config/AdapterMessageBindings.json)
where /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10 is the kamanja-140 location after easyInstallKamanja.sh

==================================
... in the documentation there should be better explanation for the arguments.
for example: 
1) should i use ClusterConfig.json from Kamanja-133/config (old) or from Kamanja-140/ClusterInstall (new) ?
2) should $KAMANJA_HOME refer to Kamanja-133 dir? or Kamanja-140 dir?

**Comment by ligadata-haitham** [comment]
[ClusterConfig.zip](https://github.com/LigaData/Kamanja/files/247572/ClusterConfig.zip)

**Comment by ligadata-haitham** [comment]
i was able to generate the json string by passing the older ClusterConfig.json file, that is from version 113.
i think there should be better description of the parameters in the Documentation to avoid confusion.

**Comment by ligadata-alison** [comment]
Hi Ramana,
    Haitham mentions that there should be a better description of:

1) should i use ClusterConfig.json from Kamanja-133/config (old) or from
Kamanja-140/ClusterInstall (new) ?
2) should $KAMANJA_HOME refer to Kamanja-133 dir? or Kamanja-140 dir?

when we call GenerateAdapterBindings in cluster installation? Would you
please let me know these answers?
thanks,
alison

**Comment by ligadata-pokuri** [comment]
Both should point to old version. This tool looks for old version KAMANJA_HOME & ClusterConfig.json

**Comment by ligadata-haitham** [comment]
When i use the old version of kamanja "$KAMANJA_HOME" i get the following error : 
[root@localhost ClusterInstall]# ./GenerateAdapterBindings.sh --KamanjaHome $KAMANJA_HOME --ClusterConfig /opt/Kamanja/config/ClusterConfig.json --OutputFile /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10/config/AdapterMessageBindings.json

parameter is --KamanjaHome
parameter is --ClusterConfig
parameter is --OutputFile
Error: Could not find or load main class com.ligadata.Migrate.GenerateAdapterBindings
result = 1
=======================
...however, if i use the location of 1.4.0 installation after easyInstallKamanja.sh , i get the correct AdapterBindings.json file : 
./GenerateAdapterBindings.sh --KamanjaHome /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10 --ClusterConfig /opt/Kamanja/config/ClusterConfig.json --OutputFile /opt/KamanjaDoubleVersions2/Kamanja-1.4.0_2.10/config/AdapterMessageBindings.json

=======================
to summarize : 
1) use older ClusterConfig.json file for --ClusterConfig
2) use location of Kamanja-140 after easyInstallKamanja.sh for --KamanjaHome

**Comment by ligadata-haitham** [comment]
Generating the adapterBinding is successful, no further issue to report.
